src/test/java/life/qbic/openbis/openbisclient/OpenBisClientMock.java

```diff
@@ -472,4 +473,10 @@ public InputStream getDatasetStream(String datasetCode, String folder) {
     return OpenBisClientHelper.getPropertiesOfEntityType(type);
   }
 
+  @Override
+  public List<DataSetFile> getFilesOfDataSetWithID(String permID) {
+    // TODO Auto-generated method stub
+    return null;
+  }
+
```

Comment:
Is that intended?

Reply:
for now, yes, this is only the mock client and it returns null for most of the methods.